src/helpers/figures/charts/pie_chart.ts

```diff
@@ -216,25 +225,93 @@ function getPieConfiguration(
   config.options.layout = {
     padding: { left: 20, right: 20, top: chart.title ? 10 : 25, bottom: 10 },
   };
-  config.options.plugins!.tooltip!.callbacks!.title = function (tooltipItems) {
-    return tooltipItems[0].dataset.label;
-  };
   config.options.plugins!.tooltip!.callbacks!.label = function (tooltipItem) {
     const { format, locale } = localeFormat;
     const data = tooltipItem.dataset.data;
     const dataIndex = tooltipItem.dataIndex;
     const percentage = calculatePercentage(data, dataIndex);
 
-    const xLabel = tooltipItem.label || tooltipItem.dataset.label;
     const yLabel = tooltipItem.parsed.y ?? tooltipItem.parsed;
     const toolTipFormat = !format && yLabel > 1000 ? "#,##" : format;
     const yLabelStr = formatValue(yLabel, { format: toolTipFormat, locale });
 
-    return xLabel ? `${xLabel}: ${yLabelStr} (${percentage}%)` : `${yLabelStr} (${percentage}%)`;
+    return `${yLabelStr} (${percentage}%)`;
+  };
+  config.options.onHover = onHoverPieChart;
+  //@ts-ignore
```

Comment:
If someone has a way to do this without a ts-ignore, I'm in !

Reply:
```suggestion
  config.options.plugins!["eventPlugin"] = {
    events: ["mouseout"],
  };
```

if you want. But that's also just a way to ignore the typing 🤷 

Maybe add a comment on what aa eventPlugin even does 😅